index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <main>
+            <p class="paragraph_one">About <br>Myself</p>
+            <ul class="paragraph">
+                <li class="list_item_one">Hello I’m a software developer!
+                    I can help you build a product ,
+                    feature or website Look through some of
+                    my work and experience! If you like what
+                    you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </li>
+                <li class="list_item_two">LET'S CONNECT</li>
+            </ul>
+            <div class="social-media-icons">
+                <ul class="icons">
+                    <li><img src="images/twitter.png" alt="twitter"></li>
+                    <li><img src="images/linkedin.png" alt="linked in"></li>
+                    <li><img src="images/mid.png" alt="mid"></li>
+                    <li><img src="images/git.svg" alt="git"></li>
+                    <li><img src="images/fin.svg" alt="fing"></li>
```

Comment:
- [ ] Social icons are hyperlink elements, for them to work please make sure to wrap the `img` tag within an `a` tag